app/src/main/java/io/homeassistant/companion/android/database/DataBaseHelper.kt

```diff
@@ -0,0 +1,87 @@
+package io.homeassistant.companion.android.database
+
+import android.content.ContentValues
+import android.content.Context
+import android.database.sqlite.SQLiteDatabase
+import android.database.sqlite.SQLiteOpenHelper
+import android.util.Log
+import java.lang.Exception
+
+private const val DATABASE_VERSION = 1
+private const val DATABASE_NAME = "HOME_ASSISTANT.db"
+private const val TABLE_NAME = "Authentication_List"
+private const val COLUMN_HOST = "host"
+private const val COLUMN_USERNAME = "username"
+private const val COLUMN_PASSWORD = "password"
+
+class DataBaseHandler(context: Context, factory: SQLiteDatabase.CursorFactory?) :
+    SQLiteOpenHelper(context, DATABASE_NAME, factory, DATABASE_VERSION) {
```

Comment:
We are going to want to use [Room](https://developer.android.com/training/data-storage/room) long term rather than raw SQL, if you are comfortable taking a first stab setting that up go for it.  If not we can merge this when ready and I can change it over before we promote it.

Reply:
Replaced SQLite by Room